readme.txt

```diff
@@ -47,6 +51,13 @@ To report a security issue, please visit the [WordPress HackerOne](https://hacke
 2. U2F Security Keys section under User Profile.
 3. Email Code Authentication during WordPress Login.
 
+== Upgrade Notice ==
+
+<!-- Keep this in sync with the next version up until 1.0.0. -->
+= 0.10.0 =
+
+From version 1.0.0, this plugin will support WordPress versions up to one year old and the minimum PHP version they require.

```

Comment:
We use the built-in WP plugin upgrade notice UI to inform users about the upcoming change.

Reply:
The majority of users will never see such a message, whether it be in the Changelog, Description, or Upgrade Notice (which is seen only on the Dashboard -> Updates page)